cves/CVE-2015-8662.yml

```diff
@@ -167,15 +181,23 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 
+    Could not find exactly how the vulnerability was discovered. VideoLAN
+    states that the vulnerability was discovered by Mateusz "j00ru" Jurczyk and 
```

Comment:
Based on the findings from my CVE, Mateus and Gynvael are both developers on this project